nuget/Microsoft.ML.OnnxRuntimeGenAI.Managed.nuspec

```diff
@@ -19,7 +19,7 @@
   </metadata>
   <files>
     <file src="..\LICENSE" target="LICENSE" />
-    <file src="..\README.md" target="README.md" />
+    <file src="PACKAGE.md" target="PACKAGE.md" />
```

Comment:
Considering the location of the [python package description](https://github.com/microsoft/onnxruntime-genai/blob/main/src/python/package_description.md), shall we add this package md file to https://github.com/microsoft/onnxruntime-genai/tree/main/src/csharp?

Reply:
Good idea. I can move it

Reply:
Moved